src/core/auth.ts

```diff
@@ -22,10 +26,10 @@ export interface AuthSDK {
   ) => Promise<FetchResult<RefreshTokenMutation>>;
 }
 
-export const auth = ({
-  apolloClient: client,
-  channel,
-}: CoreMethodsProps): AuthSDK => {
+export const auth = (
```

Comment:
Leave it as it was, passing an object is more future proof if we add any additional parameters here.